test/unit/utils/net-utils.spec.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+const NetUtils = require('../../../lib/utils/net-utils')
+const connect = require('connect')
+const http = require('http')
```

Comment:
might use `net` server, it's not so heavy

Reply:
Good point, fixed